test-data/pretty_format_yaml/multi-doc-pretty-formatted.yaml

```diff
@@ -0,0 +1,7 @@
+root:
+  test:
+  - 1
+---
+second_root:
+  test:
+  - 2
```

Comment:
In case of multi-doc YAML files I think that having the explicit document start (`---`) would be beneficial.

Reply:
I thought about this, but I was on the fence because the library doesn't include it with `dump_all()`.  Also, I've seen files with and without it.  What do you think about making this configurable?  Something like "--omit-document-start" to leave it out.

Reply:
I would say that adding configurations would not really be helpful and anyway this is not a blocking issue.

By the way, https://github.com/macisamuele/language-formatters-pre-commit-hooks/pull/3#discussion_r418655126 requires some changes that most probably will move you away from using `dump_all` ;)